Tests/Common/DisposableValue.cs

```diff
@@ -9,13 +9,17 @@ namespace LoRaWan.Tests.Common
 
     public sealed class DisposableValue<T> : IDisposable
```

Comment:
[nit] Can we add some docs explaining when this util class should be used (disposing related objects to the type T but not dispose explicitly the type T, if I understand correctly).